config/filter.d/nginx-http-auth.conf

```diff
@@ -4,7 +4,8 @@
 [Definition]
 
 
-failregex = ^ \[error\] \d+#\d+: \*\d+ user "\S+":? (password mismatch|was not found in ".*"), client: <HOST>, server: \S*, request: "\S+ \S+ HTTP/\d+\.\d+", host: "\S+"(, referrer: "\S+")?\s*$
+failregex = ^ \[error\] \d+#\d+: \*\d+ user \S*:? (password mismatch(: .*)?|(was )?not found(:| in) .*), client: <HOST>, server: \S*, request: "\S+ \S+ HTTP/\d+\.\d+", host: "\S+"(, referrer: "\S+")?\s*$
```

Comment:
> this new greedy catch-all

I found a way to remove it :)

> I mean someone that need this (catch self generated message) can for example very easy add such regex

Goal is to have them by default so that application developers adopt them / can use them as their default auth failure messages. Let's say a best-practice :)